### PR TITLE
Update parse_options

### PR DESCRIPTION
**Pull request by gergocs**
JerryScript-DCO-1.0-Signed-off-by: Gergo Csizi csgergo92@gmail.com

### CODE REVIEW

File: deps/jerry/api.cc [hunk 1]
```diff
@@ -1065,20 +1067,21 @@ MaybeLocal<UnboundScript> ScriptCompiler::CompileUnboundScript(
       }
   }
 
+  jerry_value_t file_name_jerry = jerry_create_string_from_utf8((const jerry_char_t*)*file_name);
```
Comment (zherczeg) [inline deps/jerry/api.cc]: There is no need to do these conversions, the jerry value stored in source->resource_name should be passed directly.

File: deps/jerry/v8jerry/v8jerry_utils.cpp [hunk 2]
```diff
@@ -27,14 +27,17 @@ jerry_value_t JerryPolyfill::Call(const jerry_value_t this_arg, const jerry_valu
 }
 
 jerry_value_t JerryPolyfill::BuildMethod(const char* name, const char* fn_args, const char* fn_body) {
+    jerry_value_t file_name = jerry_create_string_from_utf8(reinterpret_cast<const jerry_char_t*>(name));
```
Comment (zherczeg) [inline deps/jerry/v8jerry/v8jerry_utils.cpp]: I think `jerry_create_string` is enough, since all names are ascii.

**Comment by gergocs** [comment]
Patch is ready for review

**Comment by gergocs** [comment]
Patch is ready for review

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**